curiouslearning/FeedTheMonsterJS
# Dev optimization to Develop Code merge

# Changes
- index.html - change canvas to div for feedbacktext
- index.css - added custom animation for feedback text effect
- src/common/utils.ts - added reusable method for hiding and showing elements
- src/components/feedback-particle-effect/feedback-text-effects.ts
- src/common/global-variables.ts - removed unused code/file
- src/components/feedback-particle-effect/feedback-text-worker.ts - removed unused code/file
- src/components/feedback-particle-effect/text-particle.ts - delete unused file
- src/data/feedback-fonts.ts - added support in case specific language dont support atma font

# How to test
- Play any level and make sure the feedback text effect will appear when selecting a correct stone.

Ref: [JIRA-CODE](https://curiouslearning.atlassian.net/browse/FM-69)
